src/adapter/aws-lambda/handler.ts

```diff
@@ -471,7 +471,10 @@ export const getProcessor = (event: LambdaEvent): EventProcessor<LambdaEvent> =>
 }
 
 const isProxyEventALB = (event: LambdaEvent): event is ALBProxyEvent => {
-  return Object.hasOwn(event.requestContext, 'elb')
+  if (event.requestContext) {
+    return Object.hasOwn(event.requestContext, 'elb')
+  }
+  return false
```

Comment:
This line is not covered

Reply:
That's from the report before I updated the PR with the tests. After adding the tests, the coverage report has not run as it's awaiting maintainer approval.

I ran the coverage locally (`bun coverage`) after adding the tests and it's all green.
<img width="671" alt="hono-lambda-coverage" src="https://github.com/user-attachments/assets/246d0f31-7f6c-4c1f-a503-808d854523e8">

Reply:
I see.
New tests are not executed since approval needed